public/demos/circle.slang

```diff
@@ -45,7 +48,7 @@ float4 imageMain(uint2 dispatchThreadID, int2 screenSize)
 	// draw color beam
 	uv = (2.0 * uv) - 1.0;
 	float beamWidth = (0.7 + 
-        0.5 * cos(uv.x * 10.0 * tau * 0.15 * clamp(floor(5.0 + 10.0 * cos(time)), 0.0, 10.0)))
+        0.5 * cos(uv.x * 10.0 * tau * 0.15 * clamp(floor(5.0 + 10.0 * cos(time/60.f)), 0.0, 10.0)))
```

Comment:
Is this supposed to be frame? Otherwise why are you changing it?

Reply:
hrm, yes, supposed to be frame and not time here... good catch.